Alberto-todo-list/src/App.jsx

```diff
@@ -1,17 +1,16 @@
-import React, {useState} from 'react';
```

Comment:
from react .... not sure 16? on... we don't need to import explicitly React itself and we can just import the functions we need.